### PR TITLE
First draft of my guide introduction.

### PR DESCRIPTION
**Pull request by quaid**
## Editorial Checklist

- [ ] Material submitted is a complete draft
- [ ] Authors have submitted text in English
- [ ] Authors have submitted all art and indicated proper attribution
- [ ] Material passes a basic plagiarism check

## Author Comments

- I'm unsure that the 'method to our madness' is sensitive toward mental health, but I like identifying at the beginnign that we feel there is more to our best practices than meets the eye;
- This chapter currently also serves to explain/inspire writers for the guide to have a better handle on what needs to be in a chapter.
- The PIE idea carried from the 1.0 guide may not be a good fit, but I think it's at least a good mnemonic for making sure a chapter is complete.

### CODE REVIEW

**Comment by quaid** [comment]
I've got one edit you want to watch for -- I continued to use my upside-down/reverse use of section as outside/super over chapters. This is the introduction though that gets to clarify our word usage. Hoping one of the editors has clearer language than I've come up with so far ....

**Comment by semioticrobotic** [comment]
Oops. I misspoke earlier when I said that "chapters contain sections." Chapters contain topics. Sections contact chapters. My mistake! I'm sure you're good here. :sweat_smile:

File: presenting-tosw.adoc [hunk 1]
```diff
@@ -1,41 +1,80 @@
...
-'Details of how we see growing a community via growing users, lowering barriers, and enabling people to participte and contribute at all levels. How the guide reflects this structure.'
+A key element of all this is recognizing the value in contributions of all types, at all levels.
+Rather than valorizing only a single type of contribution, the best practice is to treat all manner of gifts to the project as equal.
+These contributions comprise content and code, hours and effort, and all manner of broad contributons for the project via events, discussion forums, infrastructure maintenance, and so on.
 
-=== Structure of this guide
+When you lower barriers to participation, you make your project welcoming at all levels.
+This doesn't mean you _remove all barriers_—just lower most of them appropriately.
+Appropriate barriers are necessary for someone receiving keys to systems, for example, and these barriers are different from those allowing only certain participants to contribute a help article or a patch to the configuration management system.
+
```
Comment (semioticrobotic) [inline presenting-tosw.adoc:50]: I feel like this section just sort of ... stops. Maybe it would benefit from some sort of closing sentiment?

**Review by semioticrobotic — CHANGES_REQUESTED** [review]

**Comment by quaid** [comment]
I didn't realize this was still open and I committed changes against the older version. I'll look at this tomorrow and see what needs to be done; I think it's going to be a manual merge and rewrite.

File: presenting-tosw.adoc (hunk 1 above)
Comment (quaid) [reply presenting-tosw.adoc:50]: Should be fixed in the current merging of my messes

**Comment by quaid** [comment]
Merging into one file so that it can be reviewed by another editor for copyedit

**Pull request merged**